src/Collection.ts

```diff
@@ -309,19 +310,20 @@ export function createCollectionMixin<TBase extends Constructor>(Base: TBase) {
     /**
      * Given a bounding box, return all the objects of the collection that are contained in the bounding box.
      * If `includeIntersecting` is true, return also the objects that intersect the bounding box as well.
+     * This is meant to work with selection. Is not a generic method.
```

Comment:
Why? I disagree. I was turned into a generic method

Reply:
it isn't because it rely on something that is specific to the selection since it reads object.selectable.